pymdb.py

```diff
@@ -5,93 +5,85 @@
 
 
 def top_250():
-    """Pulls out movie names from the Imdb Top 250 webpage and adds them to a dictionary with its position.
+    """Pulls out movie names from the Imdb Top 250 webpage and adds them to a
+    dictionary with its position.
     Returns dictionary"""
 
-    movielist=dict()
-    i=1
-    url='http://www.imdb.com/chart/top'
-    file= urllib.urlopen(url).read()
-    link = re.findall('title=".*?dir.*?>(.*?)</a>',file)
+    movielist = dict()
+    i = 1
+    url = 'http://www.imdb.com/chart/top'
+    file = urllib.urlopen(url).read()
+    link = re.findall('title=".*?dir.*?>(.*?)</a>', file)
     for links in link:
-        
-        movielist[i]=links
-        i+=1
+
+        movielist[i] = links
+        i += 1
     return movielist
 
+
 def top250_yearcount():
-    """Creates a dictionary of number of movies of a particular year in the Imdb Top 250.
+    """Creates a dictionary of number of movies of a particular year in the
+    Imdb Top 250.
     Returns a dictionary"""
 
-    i=1
-    yearcount=dict()
-    
-    url='http://www.imdb.com/chart/top'
-    file= urllib.urlopen(url).read()
+    i = 1
+    yearcount = dict()
 
-            
-         
-    years = re.findall('secondaryInfo">\((.*?)\)</span>',file)
+    url = 'http://www.imdb.com/chart/top'
+    file = urllib.urlopen(url).read()
+
+    years = re.findall('secondaryInfo">\((.*?)\)</span>', file)
     for year in years:
-        
-        yearcount[year] = yearcount.get(year,0) + 1
+        yearcount[year] = yearcount.get(year, 0) + 1
     return yearcount
 
 
-def years_top250() :
-    """Creates a dictionary of year of the movies of the top 250 in the order of ranking.
+def years_top250():
+    """Creates a dictionary of year of the movies of the top 250 in the order
+    of ranking.
     Returns  dictionary"""
 
-    i=1
-    year250=dict()
-    
-    url='http://www.imdb.com/chart/top'
-    file= urllib.urlopen(url).read()
+    i = 1
+    year250 = dict()
+
+    url = 'http://www.imdb.com/chart/top'
+    file = urllib.urlopen(url).read()
 
-            
-         
-    years = re.findall('secondaryInfo">\((.*?)\)</span>',file)
+    years = re.findall('secondaryInfo">\((.*?)\)</span>', file)
     for year in years:
-        year250[i]=year
-        i+=1
+        year250[i] = year
+        i += 1
     return year250
 
-def top250_id() :
+
+def top250_id():
     """Pulls out ImDb Title ID's from the Imdb Top 250 webpage and adds them to a list.
     Returns List"""
-
-    
-
-    
-    movielist= []
-
-   
-
-    url2='http://www.imdb.com/chart/top'
-    file= urllib.urlopen(url2).read()
-    links = re.findall('<div class=".*?tconst="(.*?)"></div>',file)
-
+    movielist = []
+    url2 = 'http://www.imdb.com/chart/top'
+    file = urllib.urlopen(url2).read()
+    links = re.findall('<div class=".*?tconst="(.*?)"></div>', file)
 
     for link in links:
         movielist.append(link)
     return movielist

```

Comment:
I know I just realized. I think back then for some reason I thought re.findall returned an iterable. Will change this.